test/unittest/dynamic_types/DynamicTypes_4_2_Tests.cpp

```diff
@@ -117,9 +117,9 @@ TEST_F(DynamicTypes_4_2_Tests, Bitset)
     mybitset.d(-1);                       // 1111111111111111
     mybitset.e(5555);                     // 0001010110110011 (The most significant bit is not stored)
     mybitset.f(3851);
-    ASSERT_TRUE(mybitset.d() == 4095);    // 0000111111111111 (2^12 - 1)
-    ASSERT_TRUE(mybitset.e() == 1459);    // 0000010110110011 (The most significant bit is not stored)
-    ASSERT_TRUE(mybitset.f() == 3851);
+    ASSERT_TRUE(mybitset.d() == 31);    // 0000111111111111 (2^12 - 1)
+    ASSERT_TRUE(mybitset.e() == 19);    // 0000010110110011 (The most significant bit is not stored)
+    ASSERT_TRUE(mybitset.f() == 11);
```

Comment:
These values change because bitsets are not correctly serialized by current dynamic data API? If so, maybe instead of modifying it makes sense to comment and fix in the dynamic data API PR.

Reply:
These are the real expected values.

Reply:
I am going to rely on your knowledge. Nevertheless, these tests might be updated with the new dynamic types API.